streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/HighAvailabilityTaskAssignorTest.java

```diff
@@ -125,89 +296,10 @@ public void shouldComputeNewAssignmentIfActiveTasksWasNotOnCaughtUpClient() {
         // we'll warm up task 0_0 on client1 because it's first in sorted order,
         // although this isn't an optimal convergence
         assertThat(probingRebalanceNeeded, is(true));
-    }
-
-    @Test
-    public void shouldComputeBalanceFactorAsDifferenceBetweenMostAndLeastLoadedClients() {
```

Comment:
Since the computeBalanceFactor method was only used by these tests, I removed both it and the tests.